### PR TITLE
Feature/new school form UI

### PR DESCRIPTION
**Pull request by TimOsse**

### CODE REVIEW

File: frontend/src/components/institute/NewSchoolPage.vue [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+      this.$refs.newSchoolOpenDateFilter.save(date);//TODO
+    },
+    closeNewSchoolPage() {
+      this.resetForm();
+      this.$emit('newSchool:closeNewSchoolPage');
+    },
+    addNewSchool() {
+      this.processing = true;
+
+      ApiService.apiAxios.post(`${Routes.institute.ROOT_ENDPOINT}/????`, this.newSchool)
```
Comment (cditcher) [inline frontend/src/components/institute/NewSchoolPage.vue:483]: URL looks incorrect... is the /???? a placeholder for a future end-point?

File: frontend/src/components/institute/NewSchoolPage.vue [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+
+    allowedFacilityTypeCodesForSchoolCategoryCode(){
+      if(this.newSchool?.categoryCode){
+        return this.schoolCategoryFacilityTypesMap[this.newSchool?.categoryCode]?.map(schoolCatFacilityTypeCode =>  this.facilityTypeCodes.find(facTypCode=> facTypCode.facilityTypeCode === schoolCatFacilityTypeCode));
+      }else {
+        return [];
+      }
+    },
+  },
+  created() {
```
Comment (cditcher) [inline frontend/src/components/institute/NewSchoolPage.vue:448]: created() method is the duplicate from SchoolDetails.vue. Not sure this in itself is an issue though.

File: frontend/src/components/institute/NewSchoolPage.vue (hunk 1 above)
Comment (TimOsse) [reply frontend/src/components/institute/NewSchoolPage.vue:483]: This ticket was just for the front end UI work. I left the url like that so its obvious for whoever does the backend

File: frontend/src/components/institute/NewSchoolPage.vue (hunk 2 above)
Comment (TimOsse) [reply frontend/src/components/institute/NewSchoolPage.vue:448]: The SchoolDetails.vue allows the user to edit the school details so its form needs the same values in the dropdowns as this one. When the user creates a new school its from the school list screen so I don't see how this duplication can be avoided.

**Review by cditcher — APPROVED** [review]

**Pull request merged**